### PR TITLE
Fix debugger for BString related runtime changes

### PR DESCRIPTION
**Pull request by NipunaRanasinghe**
## Purpose

Fixes https://github.com/ballerina-platform/ballerina-lang/issues/26020 - Error when invoking functions/object methods with string args.

## Remarks
Please merge after https://github.com/ballerina-platform/ballerina-lang/pull/26018

## Check List 
- [x] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [x] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

File: misc/debug-adapter/modules/debug-adapter-core/src/main/java/org/ballerinalang/debugadapter/evaluation/EvaluationUtils.java [hunk 1]
```diff
@@ -105,7 +109,31 @@ public static BExpressionValue make(SuspendedContext context, double val) {
...
+     * Converts the user given string literal into an {@link org.ballerinalang.jvm.api.values.BString} instance.
+     *
+     * @param context suspended debug context
+     * @param val     string value
+     * @return {@link org.ballerinalang.jvm.api.values.BString} instance
+     */
+    private static Value getAsBString(SuspendedContext context, String val) throws EvaluationException {
+        List<ReferenceType> cls = context.getAttachedVm().classesByName(B_STRING_UTILS_CLASS);
+        if (cls.isEmpty()) {
+            cls = Collections.singletonList(loadClass(context, B_STRING_UTILS_CLASS, "fromString"));
```
Comment (Dilhasha) [inline misc/debug-adapter/modules/debug-adapter-core/src/main/java/org/ballerinalang/debugadapter/evaluation/EvaluationUtils.java:128]: We can use the FROM_STRING_METHOD constant here too.

File: misc/debug-adapter/modules/debug-adapter-core/src/main/java/org/ballerinalang/debugadapter/variable/VariableUtils.java [hunk 2]
```diff
@@ -203,10 +203,10 @@ static boolean isJson(Value value) {
         return Optional.of(methods.get(0));
     }
 
-    private static String removeRedundantQuotes(String str) {
+     public static String removeRedundantQuotes(String str) {
```
Comment (Dilhasha) [inline misc/debug-adapter/modules/debug-adapter-core/src/main/java/org/ballerinalang/debugadapter/variable/VariableUtils.java:206]: is there a small formatting issue?

File: misc/debug-adapter/modules/debug-adapter-core/src/main/java/org/ballerinalang/debugadapter/evaluation/EvaluationUtils.java (hunk 1 above)
Comment (NipunaRanasinghe) [reply misc/debug-adapter/modules/debug-adapter-core/src/main/java/org/ballerinalang/debugadapter/evaluation/EvaluationUtils.java]: Yeah seems I've missed it somehow. Fixed with https://github.com/ballerina-platform/ballerina-lang/pull/26050/commits/32fbe2ec4c6d0b5ef068996df00abd3b4c245a61. Thanks!

File: misc/debug-adapter/modules/debug-adapter-core/src/main/java/org/ballerinalang/debugadapter/variable/VariableUtils.java (hunk 2 above)
Comment (NipunaRanasinghe) [reply misc/debug-adapter/modules/debug-adapter-core/src/main/java/org/ballerinalang/debugadapter/variable/VariableUtils.java]: Yes it is. Fixed with 32fbe2e

**Review by Dilhasha — APPROVED** [review]

**Pull request merged**